2024-03-07 09:53:17,955 INFO    MainThread:62633 [wandb_setup.py:_flush():76] Current SDK version is 0.16.3
2024-03-07 09:53:17,955 INFO    MainThread:62633 [wandb_setup.py:_flush():76] Configure stats pid to 62633
2024-03-07 09:53:17,955 INFO    MainThread:62633 [wandb_setup.py:_flush():76] Loading settings from /home/ubuntu/.config/wandb/settings
2024-03-07 09:53:17,955 INFO    MainThread:62633 [wandb_setup.py:_flush():76] Loading settings from /home/ubuntu/Kpca/Attack/Scripts/wandb/settings
2024-03-07 09:53:17,955 INFO    MainThread:62633 [wandb_setup.py:_flush():76] Loading settings from environment variables: {'api_key': '***REDACTED***'}
2024-03-07 09:53:17,955 INFO    MainThread:62633 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-03-07 09:53:17,956 INFO    MainThread:62633 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program_relpath': 'Attack/main.py', 'program_abspath': '/home/ubuntu/Kpca/Attack/main.py', 'program': '/home/ubuntu/Kpca/Attack/main.py'}
2024-03-07 09:53:17,956 INFO    MainThread:62633 [wandb_init.py:_log_setup():526] Logging user logs to /home/ubuntu/Kpca/Attack/Scripts/wandb/run-20240307_095317-5kmpqv1u/logs/debug.log
2024-03-07 09:53:17,956 INFO    MainThread:62633 [wandb_init.py:_log_setup():527] Logging internal logs to /home/ubuntu/Kpca/Attack/Scripts/wandb/run-20240307_095317-5kmpqv1u/logs/debug-internal.log
2024-03-07 09:53:17,956 INFO    MainThread:62633 [wandb_init.py:init():566] calling init triggers
2024-03-07 09:53:17,956 INFO    MainThread:62633 [wandb_init.py:init():573] wandb.init called with sweep_config: {}
config: {}
2024-03-07 09:53:17,956 INFO    MainThread:62633 [wandb_init.py:init():616] starting backend
2024-03-07 09:53:17,956 INFO    MainThread:62633 [wandb_init.py:init():620] setting up manager
2024-03-07 09:53:17,959 INFO    MainThread:62633 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2024-03-07 09:53:17,961 INFO    MainThread:62633 [wandb_init.py:init():628] backend started and connected
2024-03-07 09:53:17,966 INFO    MainThread:62633 [wandb_init.py:init():720] updated telemetry
2024-03-07 09:53:18,005 INFO    MainThread:62633 [wandb_init.py:init():753] communicating run to backend with 90.0 second timeout
2024-03-07 09:53:18,806 WARNING MainThread:62633 [wandb_init.py:init():1198] interrupted
Traceback (most recent call last):
  File "/home/ubuntu/.conda/envs/torch1/lib/python3.8/site-packages/wandb/sdk/wandb_init.py", line 1176, in init
    run = wi.init()
  File "/home/ubuntu/.conda/envs/torch1/lib/python3.8/site-packages/wandb/sdk/wandb_init.py", line 756, in init
    result = run_init_handle.wait(
  File "/home/ubuntu/.conda/envs/torch1/lib/python3.8/site-packages/wandb/sdk/lib/mailbox.py", line 283, in wait
    found, abandoned = self._slot._get_and_clear(timeout=wait_timeout)
  File "/home/ubuntu/.conda/envs/torch1/lib/python3.8/site-packages/wandb/sdk/lib/mailbox.py", line 130, in _get_and_clear
    if self._wait(timeout=timeout):
  File "/home/ubuntu/.conda/envs/torch1/lib/python3.8/site-packages/wandb/sdk/lib/mailbox.py", line 126, in _wait
    return self._event.wait(timeout=timeout)
  File "/home/ubuntu/.conda/envs/torch1/lib/python3.8/threading.py", line 558, in wait
    signaled = self._cond.wait(timeout)
  File "/home/ubuntu/.conda/envs/torch1/lib/python3.8/threading.py", line 306, in wait
    gotit = waiter.acquire(True, timeout)
KeyboardInterrupt
2024-03-07 09:53:23,964 WARNING MsgRouterThr:62633 [router.py:message_loop():77] message_loop has been closed
